indexer/src/main/resources/config_files/portal_records_index_schema.json

```diff
@@ -78,6 +78,7 @@
           }
         }
       },
+      "sci:citation": { "type": "text" },
```

Comment:
Is this schema factor back into the ogcapi test?

Reply:
Thank you! The pr of ogcapi is here: https://github.com/aodn/ogcapi-java/pull/67 
Test is running. Please feel free to review it